content/en/checkout/reference.md

```diff
@@ -38,58 +39,58 @@ An object with the following properties:
     - `splits`: optional `Array` of `Object`s used in split payments. Each `Object` has the following properties:
       - `split_key`: `string` (<= 50 characters) with the merchant's internal key for identifying the split.
       - `split_descriptive`: `string` (<= 255 characters) with the description of the split.
-      - `value` (*required*): `number` the split funds, rounded to 2 decimals.
+      - `value` (_required_): `number` the split funds, rounded to 2 decimals.
```

Comment:
Why these changes?

Reply:
It was the linter probably. It's the same output just didn't know that the linter was so aggressive on that 🤔